account_cutoff_accrual_base/tests/test_account_cutoff.py

```diff
@@ -0,0 +1,28 @@
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountCutoff(TransactionCase):
+
+    def test_inherit_default_cutoff_account_id(self):
+        company = self.env.user.company_id
+        default_accrued_expense_account_id = \
+            company.default_accrued_expense_account_id.id or False
+        default_accrued_revenue_account_id = \
+            company.default_accrued_revenue_account_id.id or False
+
+        account_id = \
+            self.env['account.cutoff'].with_context(type='accrued_expense').\
+            _inherit_default_cutoff_account_id()
```

Comment:
`_inherit_default_cutoff_account_id` should be removed. See review on cutoff base module #61